ArtU/ArtU/Views/LessonView.swift

```diff
@@ -6,40 +6,51 @@
 //
 
 // Mock Data
-var mockLessons = ["Lesson 1", "Lesson 2", "Lesson 3", "Lesson 4"]
+struct Lesson: Identifiable, Hashable {
+    var id = UUID()
+    let name: String
+}
+let mockLessons: [Lesson] = [
+    .init(name: "Lesson 1"),
+    .init(name: "Lesson 2"),
+    .init(name: "Lesson 3"),
+    .init(name: "Lesson 4")
+]
 
 import SwiftUI
+import WebKit
+import UIKit
+
+class ViewController: UIViewController, WKUIDelegate {
+    var webView: WKWebView!
+      override func viewDidLoad() {
+         super.viewDidLoad()
+         let myURL = URL(string:"https://www.apple.com")
+         let myRequest = URLRequest(url: myURL!)
+         webView.load(myRequest)
+      }
+    override func loadView() {
+        let webConfiguration = WKWebViewConfiguration()
```

Comment:
Should webConfiguration be constant? if not use var

Reply:
The webConfiguration should be constant for each new view. The same webConfiguration instantiation will not exist across multiple views.